setup.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup
+from os import path
+
+
+UTILS_CV = "utils_cv"  # Utility folder name
+# UTILS_CV_PATH = path.join(path.abspath(path.dirname(__file__)), UTILS_CV)
+# README = path.join(UTILS_CV_PATH, "README.md")
+README = path.join(UTILS_CV, "README.md")
+exec(open(path.join(UTILS_CV, "__init__.py")).read())
+
+with open(README, encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="utils_cv",
+    version=__version__,
+    description="Computer Vision Utilities",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/ComputerVision",
+    project_urls={
+        "Bug Tracker": "https://github.com/microsoft/ComputerVision/issues",
+        "Source Code": "https://github.com/microsoft/ComputerVision",
+        "Documentation": "https://github.com/microsoft/ComputerVision",
+    },
+    author="CVDev Team at Microsoft",
+    classifiers=[
+        "Development Status :: 2-Pre-Alpha",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    include_package_data=True,
+    # Not fully test the dependency coverage, please make sure to sync
+    # with environment.yml
+    install_requires=[
+        "azureml-sdk[notebooks,contrib]>=1.0.30",  # requires ipykernel, papermill, jupyter-core, jupyter-client
+        "bqplot",
+        "fastai==1.0.48",  # requires pytorch, torchvision, nvidia-ml-py3
+        "scikit-learn>=0.19.1",
```

Comment:
Q1: How come there are only 4 libraries getting installed here, while in our environment.yaml file there are 8 pip-installable libraries?
Q2: so the user installs via "pip install git+https://github.com/microsoft/ComputerVision#egg=utils_cv" and then has to execute the "conda env create -f environment.yml" to create all dependencies?
Q3: If yes, can we just remove the 4 pip packages here. Also, if yes, this needs to be made clear in the text.

Reply:
@PatrickBue 

### A1 ### 
* Some libraries are required by these 4
  - E.g. `fast.ai` will install `pytorch`, `torchvision` and `nvidia-ml-py3`
* Some others are not depended by `utils_cv`
  - E.g. `ipywebrtc` which is used in notebook, and `black` is for reformatting.
https://github.com/microsoft/ComputerVisionBestPractices/blob/f194c2789349bf0be4d9369fe0d5cdba863bb420/environment.yml#L18-L37

### A2 ###
We can put `pip install git+https://github.com/microsoft/ComputerVision#egg=utils_cv` inside `environment.yml`.  E.g. 
<pre>
# environment.yml
name: utils_cv
channels:
- defaults
...
- dependencies:
  - pytest>=3.6.4
- pip:
  - <b>git+https://github.com/microsoft/ComputerVisionBestPractices#egg=utils_cv</b>
  - black>=18.6b4
  - ipywebrtc
  - pre-commit>=1.14.4
</pre>

### A3 ###
Yes, sure.

Reply:
@PatrickBue See 63daec9db29f3f64b9e804afb4ac091b2552ff91
https://github.com/microsoft/ComputerVision/blob/63daec9db29f3f64b9e804afb4ac091b2552ff91/environment.yml#L12-L28

Reply:
A1 sounds good. A2 see my other comments. Please undo all edits to the environment.yaml file. A3 after reverting the environment.yaml file back to its unedited version, please re-add all libraries as required.

Reply:
Done.